application/models/__init__.py

```diff
@@ -116,7 +116,7 @@ def verify_exchange_completion(exchange_id, player_id, game_id, boost_value):
 
 
 def create_player(form_data, game_id):
-    name = form_data['name']
+    name = ' '.join(map(lambda x: x[0].upper() + x[1:], form_data['name'].split()))
```

Comment:
this lambda doesn't quickly surface what it does, unless you're REAL good at reading lambdas and method chaining. comment? break up into single-steps?